### DISCUSSION TITLE
How to set the filename and file storage location?

### DISCUSSION THREAD

**Discussion by toilv** [souzatharsis/podcastfy#191] [General]
Hi,
I want to set the filename and file location but seem there is no option for it?
Is that right?

**Comment by souzatharsis** [comment]
Output directories can be defined by passing a conversation config file:

https://github.com/souzatharsis/podcastfy/blob/main/usage/conversation_custom.md#general-tts-settings

Custom file name is not supported yet. But it should be straightforward to
implement it. Feel free to submit a PR. Happy to help.

**Reply by toilv** [reply]
Thank you
